### ISSUE TITLE
Nametagged Zombified Piglins desapwn

### ISSUE THREAD

**Issue by ImToe1** [PaperMC/Paper#3747]
### Steps/models to reproduce:
___This may include a build schematic, a video, or detailed instructions to help reconstruct the issue___

### Plugin list:
![image](https://user-images.githubusercontent.com/67716722/86300645-cc7eb080-bc03-11ea-952e-72cbdccc03cf.png)

### Paper build number:
git-Paper-30 (MC: 1.16.1)

### Anything else:
___Anything else you think may help us resolve the problem___

**Comment by JRoy** [comment]
Can you reproduce this on vanilla or spigot?

**Comment by ImToe1** [comment]
it didnt happen on vanilla, only when  i updated to paper

**Comment by JRoy** [comment]
Can you reproduce it on Spigot?

**Comment by ImToe1** [comment]
i dont know, i dont have a spigot server

**Comment by ImToe1** [comment]
i have build ilmangos gold and exp farm, but updated so it works on 1.16, and it was working fine on vanilla, but when i updated to paper the zombified piglins started not aggroing properly to the player and the nametagged zombified piglins were despawn while you used the farm.

**Comment by Machine-Maker** [comment]
I cannot reproduce this on paper-79. I spawned zombified piglins in, ran `execute as @e[type=minecraft:zombified_piglin] run say "hey!"`, to keep track of the number of zombified piglins in the world. I named 1, and left one without a name. 
First time I ran the command, I was right next to them, and there were 2 chat messages, then I flew a long ways away, ran it again and say only 1 message meaning 1 despawned.
